frontend/scripts/build.sh

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env nix-shell
 #! nix-shell -j 4 -i bash -p stack git
-#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/464c79ea9f929d1237dbc2df878eedad91767a72.tar.gz
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/763e21e982370f67c126f92a1113ea949db3b6e0.tar.gz
+export NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/763e21e982370f67c126f92a1113ea949db3b6e0.tar.gz
```

Comment:
Could you write a comment where is this pointing to, what package is that? I tried looking for that, couldn't find it. I don't know how I would find this, just by searching for `763e21e982370f67c126f92a1113ea949db3b6e0` ?